src/EditorFeatures/CSharpTest/Utilities/CodeSnippets.cs

```diff
@@ -0,0 +1,49 @@
+﻿namespace Microsoft.CodeAnalysis.Editor.CSharp.UnitTests
+{
+    internal static class CodeSnippets
+    {
+        public const string FormattableStringType = @"
+namespace System
+{
+    public abstract class FormattableString : IFormattable
```

Comment:
Today, we load specific assemblies out of our test resources. Updating those to 4.6 feels out of scope for this change. (Note that the compiler tests do the same trick we're doing here.)